docs/src/components/ImageContainer.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { css } from "styled-components";
+import { defaultTheme } from "@kiwicom/orbit-components";
+
+interface Props {
+  align?: "left" | "center" | "right";
+  border?: boolean;
+  children: React.ReactNode;
+  size?: "small" | "medium" | "large";
+}
+
+const ImageContainer = ({ align = "center", border, size = "large", children }: Props) => {
+  const getContainerWidth = containerSize => {
+    if (containerSize === "small") return "200px";
+    if (containerSize === "medium") return "400px";
+    return "650px";
+  };
+  const containerWidth = getContainerWidth(size);
```

Comment:
It's a little confusing that `getContainerWidth` and `getContainerMargin` take an argument, so either move them outside of the component or use whatever value you're passing inside the function, e.g.:

```js
  const getContainerWidth = () => {
    if (size === "small") return "200px";
    if (size === "medium") return "400px";
    return "650px";
  };
  const containerWidth = getContainerWidth();
```